docs/arduino/usecase2.md

```diff
@@ -0,0 +1,58 @@
+# From constitutive parameter calibration to site response analysis
+
+#### A collection of educational notebooks to introduce model parameter calibration and site response analysis using OpenSees in DesignSafe-CI. 
+
+**Pedro Arduino - University of Washington** <br/>
+**Sang-Ri Yi - SimCenter, UC Berkeley** <br/>
+**Aakash Bangalore Satish - SimCenter, UC Berkeley**
+
+
+## Resources
+
+The example makes use of the following DesignSafe resources:
```

Comment:
Could you please directly link to the notebooks? If you have multiple notebooks you could do something like a table:

https://github.com/DesignSafe-CI/DS_Use_Case_template/blob/master/docs/brandenberg/apiusecases/usecase.md

Reply:
In your case maybe a table of 

| Description | UseCase | Notebook|
|-------|----|---|

Since you use this as a root notebook for other subnotebooks?